packages/core/src/core/adapter_format.ts

```diff
@@ -243,3 +241,44 @@ export const syncWorkspaceToFolder = ({
     'syncWorkspaceToFolder %s',
     baseDir,
   )
+
+type UpdateElementFolderArgs = {
+  workspace: Workspace
+  baseDir: string
+  accountName: string
+  changes: ReadonlyArray<Change>
+}
+
+export type UpdateElementFolderResult = {
+  errors: ReadonlyArray<SaltoError>
+}
+
+export const updateElementFolder = async ({
```

Comment:
I missed this on the first round, but this doesn't need to be `async`
```suggestion
export const updateElementFolder = ({
```

Reply:
that's a good catch